src/qtiCommonRenderer/renderers/interactions/TextEntryInteraction.js

```diff
@@ -81,10 +81,14 @@ var render = function render(interaction) {
         maxChars = parseInt(patternMaskHelper.parsePattern(patternMask, 'chars'), 10);
 
     // Setting up baseType
-    if (baseType === 'integer') {
-        $input.attr('inputmode', 'numeric');
-    } else if (baseType === 'float') {
-        $input.attr('inputmode', 'decimal');
+    if (baseType) {
```

Comment:
I think this `if() {} else if() {} else if() {} else {}` is the definition of the `switch case`

Reply:
Done :+1: